### ISSUE TITLE
Carthage build fails

### ISSUE THREAD

**Issue by phedlund** [SyncDB/Sync#377]
````
*** Building scheme "Sync-macOS" in Demo.xcodeproj
** BUILD FAILED **


The following build commands failed:
	CompileSwift normal x86_64
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
	Ditto /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework/Versions/A/Headers/Sync-Swift.h /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Intermediates/Demo.build/Release/Sync-macOS.build/Objects-normal/x86_64/Sync-Swift.h
	Ditto /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework/Versions/A/Modules/Sync.swiftmodule/x86_64.swiftmodule /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Intermediates/Demo.build/Release/Sync-macOS.build/Objects-normal/x86_64/Sync.swiftmodule
	Ditto /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework/Versions/A/Modules/Sync.swiftmodule/x86_64.swiftdoc /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Intermediates/Demo.build/Release/Sync-macOS.build/Objects-normal/x86_64/Sync.swiftdoc
	Ld /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework/Versions/A/Sync normal x86_64
	GenerateDSYMFile /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework.dSYM /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework/Versions/A/Sync
(7 failures)
/Users/user/Project/Carthage/Checkouts/Sync/Source/Sync/NSArray+Sync.swift:2:8: error: module file's minimum deployment target is OS X v10.11: /Users/user/Project/Carthage/Checkouts/Sync/Carthage/Build/Mac/DATAStack.framework/Modules/DATAStack.swiftmodule/x86_64.swiftmodule
clang: error: linker command failed with exit code 1 (use -v to see invocation)
error: cannot parse the debug map for "/Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework/Versions/A/Sync": No such file or directory
A shell task (/usr/bin/xcrun xcodebuild -project /Users/user/Project/Carthage/Checkouts/Sync/Demo.xcodeproj -scheme Sync-macOS -configuration Release ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build) failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	CompileSwift normal x86_64
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
	Ditto /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework/Versions/A/Headers/Sync-Swift.h /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Intermediates/Demo.build/Release/Sync-macOS.build/Objects-normal/x86_64/Sync-Swift.h
	Ditto /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework/Versions/A/Modules/Sync.swiftmodule/x86_64.swiftmodule /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Intermediates/Demo.build/Release/Sync-macOS.build/Objects-normal/x86_64/Sync.swiftmodule
	Ditto /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework/Versions/A/Modules/Sync.swiftmodule/x86_64.swiftdoc /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Intermediates/Demo.build/Release/Sync-macOS.build/Objects-normal/x86_64/Sync.swiftdoc
	Ld /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework/Versions/A/Sync normal x86_64
	GenerateDSYMFile /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework.dSYM /Users/user/Library/Developer/Xcode/DerivedData/Demo-fesonrjvpdhxkufsnhiqdsztrrjc/Build/Products/Release/Sync.framework/Versions/A/Sync
(7 failures)
````

**Comment by 3lvis** [comment]
Hi!

Which command were you running for that to happen?

**Comment by phedlund** [comment]
```
carthage update --use-submodules
```

**Comment by 3lvis** [comment]
Should be fixed in 3.0.0

https://github.com/SyncDB/Sync/releases/tag/3.0.0
